retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/Image.java

```diff
@@ -1,24 +1,37 @@
 package com.example.aperez.retrofittest.mvp.model;
 
+import com.activeandroid.Model;
+import com.activeandroid.annotation.Column;
+import com.activeandroid.annotation.Table;
+import com.activeandroid.query.Select;
 import com.google.gson.annotations.SerializedName;
 
+import java.util.List;
+
 /**
  * Created by aperez on 24/01/17.
  */
 
-public class Image {
+@Table(name = "Images")
+public class Image extends Model{
```

Comment:
Create another class for this. The Image class is used as response to a retrofit service.